common/static/common/js/spec/components/paging_collection.js

```diff
@@ -0,0 +1,38 @@
+define(["backbone.paginator", "backbone"], function(BackbonePaginator, Backbone) {
+    // This code was adapted from collections/asset.js.
+    var PagingCollection = BackbonePaginator.requestPager.extend({
+        model : Backbone.Model,
+        paginator_core: {
+            type: 'GET',
+            accepts: 'application/json',
+            dataType: 'json',
+            url: function() { return this.url; }
+        },
+        paginator_ui: {
+            firstPage: 0,
+            currentPage: 0,
+            perPage: 50
```

Comment:
I'm not familiar with the `BackbonePaginator` plugin, but it says it's a drop-in replacement for `Backbone.Collection`. I think it'd be pretty useful to be able to override these values, possibly via the constructor.